app/src/main/java/org/oppia/android/app/drawer/Argument.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.android.app.drawer
+
+sealed class Argument {
```

Comment:
Why do we need a sealed class if we have a proto with a oneof? That should satisfy the same need.